app/webpack/reactjs/components/mau_button.tsx

```diff
@@ -1,18 +1,27 @@
+import * as types from "@reactjs/types";
 import cx from "classnames";
 import React, { FC } from "react";
 
 interface MauButtonProps {
   type?: "button" | "submit";
   primary?: boolean;
   secondary?: boolean;
+  fullWidth?: boolean;
   onClick?: (event: Event) => void;
   disabled?: boolean;
 }
 
+export const buttonStyleAttrs = (
```

Comment:
this is a funny function but I preferred it over a `variant` cuz I like the attributes "primary" "fullWidth" or whatever